styleguide/screens/Utilities/Modifiers.js

```diff
@@ -110,43 +117,56 @@ const Modifiers = () => (
         code={ dedent`
           // --size-lg-vii / 12.5rem / 200px
           .ptLgVii
+
           // --size-lg-vi / 9rem / 144px
           .ptLgVi
+
           // --size-lg-v / 6rem / 96px
           .ptLgV
+
           // --size-lg-iv / 5rem / 80px
           .ptLgIv
+
           // --size-lg-iii / 4rem / 64px
           .ptLgIii
+
           // --size-lg-ii / 3rem / 48px
           .ptLgIi
+
           // --size-lg-i / 1.5rem / 24px
           .ptLgI
+
           // --size-regular / 1rem / 16px
           .ptr
+
           // --size-sm-i / 0.75rem / 12px
           .ptSmI
+
           // --size-sm-ii / 0.5rem / 8px
           .ptSmIi
+
           // --size-sm-iii / 0.25rem / 4px
           .ptSmIii
+
           // --size-sm-iv / 0.1rem / 1.6px
           .ptSmIv
         ` }
-        syntax="css"
+        syntax="jsx"
       />
     </D>
     <D>
-      <H level={ 3 }>Colours</H>
+      <H level={ 3 }>Colors</H>
       <T elm="p" className={ m.mtr }>
-        As with the sizes, all of our colours are also available as modifiers.
-        You can apply the colours to the foreground of a background element like so:
+        As with the sizes, all of our
+        <Link className={ scaffoldCss.link } to="/design/colors">colors</Link> are also available
```

Comment:
Whitespace needed <img width="74" alt="screen shot 2017-11-08 at 11 22 27" src="https://user-images.githubusercontent.com/1636241/32546520-510c843a-c477-11e7-9e40-e9a38cdef2c3.png">

Reply:
Also we're English 🇬🇧